src/main.py

```diff
@@ -1,9 +1,21 @@
+import subprocess
+
 from follower import Follower
 from leader import Leader
 from tagrec import TagRecognition
 
+'''
+Calls hcitool and reads the exit value returned. hcitool returns 0 on success 
+and 1 on failure.
+
+Input: Bluetooth address.
+Output: Boolean values, true if connected false if disconnected.
+'''
+def is_connected(bt_addr):
+    return not subprocess.call(['hcitool','info', bt_addr])
```

Comment:
I have a lot of problems with this, but before that:

Can't we just check if /dev/input/js0 exists?